commands/core/views.d8.drush.inc

```diff
@@ -376,7 +376,7 @@ function _views_drush_op($op = '', array $view_names = array()) {
 
   $view_names = MapArray::copyValuesToKeys($view_names);
```

Comment:
`MapArray::copyValuesToKeys()` no longer exists. This should be `array_combine($view_names, $view_names)` now. Could you fix that as well?